ckanext/opendata_theme/opengov_custom_footer/plugin/__init__.py

```diff
@@ -0,0 +1,52 @@
+import ckan.plugins as plugins
+import ckan.plugins.toolkit as toolkit
+from ckan.exceptions import CkanVersionException
+
+try:
+    toolkit.requires_ckan_version("2.9")
+except CkanVersionException:
+    from ckanext.opendata_theme.opengov_custom_footer.plugin.pylons_plugin import MixinPlugin
+    from webhelpers.html import literal
+    from ckanext.opendata_theme.opengov_custom_footer.controller import CustomFooterController
```

Comment:
There is an import error here as controller.py was renamed. Instead this should probably be `common_controller`.

Reply:
updated.